src/components/Dao/DaoWiki.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import * as Sticky from "react-stickynode";
+import { IDAOState, ISchemeState, Scheme, IProposalType, Proposal } from "@daostack/client";
+import { WikiContainer, actualHash, ReactiveWiki } from "@dorgtech/daosmind";
+import classNames from "classnames";
+import { enableWalletProvider } from "arc";
+import { combineLatest } from "rxjs";
+
+import { Link } from "react-router-dom";
+import * as arcActions from "actions/arcActions";
+import { showNotification } from "reducers/notifications";
+import { schemeName } from "lib/util";
+import { connect } from "react-redux";
+import { RouteComponentProps } from "react-router-dom";
+import withSubscription, { ISubscriptionProps } from "components/Shared/withSubscription";
+import Loading from "components/Shared/Loading";
+import * as proposalStyle from "../Scheme/SchemeProposals.scss";
+import * as daoStyle from "./Dao.scss";
+
+type IExternalProps = {
+  daoState: IDAOState;
+  match: Record<string, any>;
+} & RouteComponentProps<any>;
+
+const mapDispatchToProps = {
+  createProposal: arcActions.createProposal,
+  showNotification,
+};
+
+interface IDispatchProps {
+  createProposal: typeof arcActions.createProposal;
+  showNotification: typeof showNotification;
+}
+
+type SubscriptionData = ISubscriptionProps<[Scheme[], Proposal[]]>
+type IProps = IDispatchProps & IExternalProps & SubscriptionData;
+
+function DaoWiki(props: IProps) {
+  const [hasWikiScheme, setHasWikiScheme] = React.useState<boolean>(false);
+  const [schemes, proposals] = props.data;
+
+  const renderWikiComponent = () => {
+    const { daoAvatarAddress, perspectiveId, pageId } = props.match.params;
+    actualHash["dao"] = daoAvatarAddress;
```

Comment:
what is actualHash?

Reply:
actualHash is a variable that will be handled by the package `@dorgtech/daosmind` the function of it is to let the wiki know which wiki perspective and/or page it should render (if it has any, since these two are optional url parameters)